Amasty_Gdpr/web/template/checkout/gdpr-consent.html

```diff
@@ -46,16 +46,19 @@
             class="checkbox__label"
             data-bind="attr: {'for': checkbox_code + $parent.getUniqId()}"
         >
-            <span 
-                lass="action" 
+            <span
+                lass="action"
                 data-bind="html: checkbox_text"
             ></span>
         </label>
     </div>
 </div>
 <!-- /ko -->
-<input 
-    hidden 
-    name="am-gdpr-checkboxes-from" 
-    data-bind="attr: {'value': metaInfo.where}"
->
+
+<!-- ko if: metaInfo -->
```

Comment:
should we check `isEnabled` ?
in js file I see:            isEnabled: gdprConfig.length  !== 0`

If isEnabled will work here then I guess we should use this instead of checking metaInfo.

Reply:
yes, thanks!